README.md

```diff
@@ -1,24 +1,91 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-app project
 
-Things you may want to cover:
+> This is a ruby on rails mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Ruby version
+# Demo Link
 
-* System dependencies
+Click [here](https://julie-budgetapp.herokuapp.com/) to view the website online.
 
-* Configuration
+# Video presentation Link
 
-* Database creation
+Click [here](#)
```

Comment:
The provided video link to this Pull Request does is invalid. Kindly endeavor to record a maximum of 5 minutes video presenting the features of the project. The recorded video should:
- [ ]  entail the **features** of the project.
- [ ] have a maximum length of **5 minutes**.
- [ ] be **clear and objective description** about the project in professional English. The topic and purpose of the project are made clear.
- [ ] contains a demo of the project, highlighting all the project features.
- [ ] contain a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
I have included the video presentation link to README. Thank